doc/styles/Vocab/ANSYS/accept.txt

```diff
@@ -69,6 +69,7 @@ GPa
 GUI
 hexahedral
 hostname
+HTML
 Imagemagick
 ImageMagick

```

Comment:
You need to simplify this accept.txt using the proper regex for your different entries. Maybe @SMoraisAnsys can give you a hand with this.

Reply:
Done in a83cec1c69547a8edd8b957a32a51e6fb35e0649

Reply:
Pinging @SMoraisAnsys for feedback